client/state/themes/actions.js

```diff
@@ -133,32 +136,30 @@ export function receiveServerError( error ) {
  * Returns an action object to be used in signalling that a theme activation
  * has been triggered
  *
- * @param  {Object}  theme Theme received
- * @param  {Object}  site Site used for activation
+ * @param  {Number}  themeId Theme to be activated
+ * @param  {Object}  siteId Site used for activation
  * @return {Object}  Action object
  */
-export function activateTheme( theme, site ) {
+export function themeActivation( themeId, siteId ) {
```

Comment:
`themeActivation` is just action creator.
`activateTheme` is still chosen name for function that activates themes :smile: